src/schema/v2/__tests__/marketingCollections.test.ts

```diff
@@ -172,4 +172,105 @@ describe("MarketingCollections", () => {
       curatedMarketingCollections: payload,
     })
   })
+
+  it("uses the curated sort for marketing collections", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CURATED, size: 2, category: "Gallery") {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
+      },
+    ]
```

Comment:
**suggestion**: These payloads don't need to be redefined. You can reference the `marketingCollectionsData` fixture defined above, which should be in scope here.

Reply:
They are within scope, but I find it helpful to ensure that the defined payload matches the expected response. This helps avoid any manipulation of the original `marketingCollectionsData` fixture. Does that make sense?

Reply:
I might agree if the fixture data were changing somehow but we're duplicating the same values and even the same count here. Without a need to specify different data, redefining it here just clutters the test and makes it harder to read.

I don't see any risk here in anything manipulating the original fixture. It's defined at it's current scope _specifically so_ it can be reused in nested spec definitions.

Reply:
Okay if it is the problem is around the cluttering of the test, I'll define that payload on the top and re-use it.